articles/introduction-to-wireshark/index.md

```diff
@@ -0,0 +1,228 @@
+### Introduction to Network Analysis Using Wireshark
+As an IT professional, one of the most powerful networking tools you will find yourself using is ```Wireshark```, a tool mostly used in the analysis of network packets.  
+In this tutorial, we will be looking at an overview of ```Wireshark```, how it works, and several other uses. 
+
+### Requirements
+* Networking Basics
+* TCP/IP stack
+* Reading and interpreting packet headers
+* Routing and port forwarding
+* DHCP
+
+### What is Wireshark?
+
+Wireshark software's history goes back to 1998, as an open-source project started by volunteer network experts around the world.  
+The main aim was to build a standard network protocol analyzing tool.  
+
+Therefore, ```Wireshark``` is simply a network packet analyzer that is mainly used to capture the data on your network. This data is then presented in a form understandable by human beings.  
+
+In case you find yourself in a position where you need to try and troubleshoot every single network packet, ``` Wireshark``` is the right tool for you!
+
+This is a very powerful tool, as it enables you, the user, to perform various operations such as:
+* Troubleshooting your network.
+* Performing security operations that are used to detect security threats such as port scanning.  
+* Learning more about your network protocols at the microscopic level.
+* Performing analysis of voice over the internet(VoIP).
+
+### How Wireshark works
+As we have said, ```Wireshark``` is a packet analyzer, sometimes other people refer to it as a packet sniffer, so do not get confused.  
+
+How it works is very simple, let's take your machine, for example, connected to your ```WIFI``` or the network you're currently using, and you want to determine what is currently going on, on your network.  
+What happens is that whenever you use ```Wireshark```, it will capture your network traffic(this refers to the data moving currently on your network) and record every single data offline.  
```

Comment:
Don't use future tense in your writing. Use present tense. Avoid it will capture. Instead, write it captures.